### ISSUE TITLE
Fix `aria-labelledby` attribute in `<ContentInfo>`

### ISSUE THREAD

**Issue by trezy** [trezy/trezy.com#166]
There are typos in the `<ContentInfo>` component. `aria-labelledBy` should be `aria-labelledby`.
